src/app/work-item/work-item-list/work-item-list.component.ts

```diff
@@ -175,12 +176,25 @@ export class WorkItemListComponent implements OnInit, AfterViewInit {
   }
 
   // event handlers
+  onToggle(entryComponent: WorkItemListEntryComponent): void {
+    // This condition is to select a single work item for movement
+    // deselect the previous checked work item
+    if (this.workItemToMove)
+      this.workItemToMove.uncheck();
```

Comment:
Can you enclose this line with {}. I understand it's just a single line statement, but enclosing increases readability.